docs/api/networksocket/SocketStats.md

```diff
@@ -0,0 +1,21 @@
+## SocketStats
+
+Use the SocketStats class to read the network socket statistics. SocketStats Class object is created by "InternetSocket" class when statistics are enabled by `nsapi.socket-stats-enable` config option.
+
+SocketStats class provides static API `mbed_stats_socket_get_each` to get the network socket information. Using this API you can know how many sockets are open, destination IPand data bytes sent and received. Also the timestamp of recent "socket state" change is recorded in `last_change_tick`.
+
+Socket Statistics are stored in intenal data buffer and maintained for closed sockets as well. In case internal data buffer is full, socket closed with oldest timestamp will be over-written. Maximum amount of sockets statistics stored can be configured by `nsapi.socket-stats-max-count` config option.
```

Comment:
Link to config page maybe @AnotherButler ?

Reply:
@kegilbert - I don't see network configs here https://github.com/ARMmbed/mbed-os-5-docs/tree/development/docs/reference/configuration

Reply:
Good point, forgot to check for that earlier. We should add that page as well. Can do it in this PR or another one.

Reply:
Will be good to do it in another one.